src/components/EboardPeople.vue

```diff
@@ -2,7 +2,33 @@
   <div>
     <div class="flexWrapper">
       <div class="row">
-        <div class="column" v-bind:key="person.id" v-for="person in eboard">
+        <div
+          class="column"
+          v-bind:key="person.id"
+          v-for="person in eboard.slice(0, 3)"
+        >
+          <div>
+            <Person v-bind:person="person" />
+          </div>
+        </div>
+      </div>
+      <div class="row">
+        <div
+          class="column"
+          v-bind:key="person.id"
+          v-for="person in eboard.slice(3, 6)"
```

Comment:
Food for thought, we could make this even more scalable if we want to by abstracting this process of pulling out three people from the array, so that we don't have to write out each row.

Reply:
💯